spec/requests/api/v1/spots/index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe 'GET api/v1/spots', type: :request do
+  let!(:user)           { create(:user) }
```

Comment:
no need of using bang here as you are instantiating the user in the created_spots let